pkg/evpn/svi.go

```diff
@@ -130,51 +164,51 @@ func (s *Server) DeleteTunnel(_ context.Context, in *pb.DeleteTunnelRequest) (*e
 		return nil, err
 	}
 	resourceID := path.Base(obj.Name)
-	// use netlink to find Subnet/Bridge
-	vrf, err := netlink.LinkByName(resourceID)
+	// use netlink to find vlan
+	vlan, err := netlink.LinkByName(resourceID)
 	if err != nil {
 		err := status.Errorf(codes.NotFound, "unable to find key %s", resourceID)
 		log.Printf("error: %v", err)
 		return nil, err
 	}
 	// bring link down
-	if err := netlink.LinkSetDown(vrf); err != nil {
+	if err := netlink.LinkSetDown(vlan); err != nil {
```

Comment:
Any reason to bring the SVI down before deletion?

Reply:
I thought this is more proper, but you are the expert

Reply:
No harm to bring it down first. I will agree with @glimchb